packages/terra-date-time-picker/tests/wdio/date-time-picker-spec.js

```diff
@@ -11,6 +11,7 @@ Terra.describeViewports('DateTimePicker', ['tiny', 'large'], () => {
   describe('Default', () => {
     before(() => {
       browser.url('/#/raw/tests/terra-date-time-picker/date-time-picker/date-time-picker-default');
+      browser.refresh();
```

Comment:
For some weird reason, I had to add this to fix the following accessibility violation. I've no clue what suddenly set the `aria-hidden` attribute on the `root` element.

<details>
 <summary>WDIO Axe Error Report</summary>

```
{
  "description": "Ensures aria-hidden elements do not contain focusable elements",
  "help": "ARIA hidden element must not contain focusable elements",
  "helpUrl": "https://dequeuniversity.com/rules/axe/3.2/aria-hidden-focus?application=axeAPI",
  "id": "aria-hidden-focus",
  "impact": "serious",
  "nodes": [
    {
      "all": [
        {
          "data": null,
          "id": "focusable-disabled",
          "impact": "serious",
          "message": "Focusable content should be disabled or be removed from the DOM",
          "relatedNodes": [
            {
              "html": "<input class=\"Input-module__form-input___2z_wq DatePicker-module__input___p3mYV\" placeholder=\"MM/DD/YYYY\" name=\"terra-date-input\" type=\"text\" value=\"\">",
              "target": [
                ".DatePicker-module__input___p3mYV"
              ]
            },
            {
              "html": "<button class=\"Button-module__button___3afIE Button-module__neutral___ZyOh8 Button-module__compact___31Uah DatePicker-module__button___1KmcA\" type=\"button\" aria-disabled=\"false\" aria-label=\"Open Calendar\" title=\"Open Calendar\">",
              "target": [
                "button"
              ]
            },
            {
              "html": "<input aria-label=\"Hours\" class=\"Input-module__form-input___2z_wq TimeInput-module__time-input-hour___2VQ6_ TimeInput-module__desktop___1gtms\" placeholder=\"hh\" maxlength=\"2\" size=\"2\" name=\"terra-time-hour-input\" type=\"text\" pattern=\"\\d*\" value=\"\">",
              "target": [
                ".TimeInput-module__time-input-hour___2VQ6_"
              ]
            },
            {
              "html": "<input aria-label=\"Minutes\" class=\"Input-module__form-input___2z_wq TimeInput-module__time-input-minute___1elGu TimeInput-module__without-second___2G5o1 TimeInput-module__desktop___1gtms\" placeholder=\"mm\" maxlength=\"2\" size=\"2\" name=\"terra-time-minute-input\" type=\"text\" pattern=\"\\d*\" value=\"\">",
              "target": [
                ".TimeInput-module__time-input-minute___1elGu"
              ]
            }
          ]
        }
      ],
      "any": [],
      "failureSummary": "Fix all of the following:\n  Focusable content should be disabled or be removed from the DOM",
      "html": "<div id=\"root\" aria-hidden=\"true\">",
      "impact": "serious",
      "none": [],
      "target": [
        "#root"
      ]
    }
  ],
  "tags": [
    "cat.name-role-value",
    "wcag2a",
    "wcag412"
  ]
}
running chrome
AssertionError: expected no accessibility violations but got:
	{
  "description": "Ensures aria-hidden elements do not contain focusable elements",
  "help": "ARIA hidden element must not contain focusable elements",
  "helpUrl": "https://dequeuniversity.com/rules/axe/3.2/aria-hidden-focus?application=axeAPI",
  "id": "aria-hidden-focus",
  "impact": "serious",
  "nodes": [
    {
      "all": [
        {
          "data": null,
          "id": "focusable-disabled",
          "impact": "serious",
          "message": "Focusable content should be disabled or be removed from the DOM",
          "relatedNodes": [
            {
              "html": "<input class=\"Input-module__form-input___2z_wq DatePicker-module__input___p3mYV\" placeholder=\"MM/DD/YYYY\" name=\"terra-date-input\" type=\"text\" value=\"\">",
              "target": [
                ".DatePicker-module__input___p3mYV"
              ]
            },
            {
              "html": "<button class=\"Button-module__button___3afIE Button-module__neutral___ZyOh8 Button-module__compact___31Uah DatePicker-module__button___1KmcA\" type=\"button\" aria-disabled=\"false\" aria-label=\"Open Calendar\" title=\"Open Calendar\">",
              "target": [
                "button"
              ]
            },
            {
              "html": "<input aria-label=\"Hours\" class=\"Input-module__form-input___2z_wq TimeInput-module__time-input-hour___2VQ6_ TimeInput-module__desktop___1gtms\" placeholder=\"hh\" maxlength=\"2\" size=\"2\" name=\"terra-time-hour-input\" type=\"text\" pattern=\"\\d*\" value=\"\">",
              "target": [
                ".TimeInput-module__time-input-hour___2VQ6_"
              ]
            },
            {
              "html": "<input aria-label=\"Minutes\" class=\"Input-module__form-input___2z_wq TimeInput-module__time-input-minute___1elGu TimeInput-module__without-second___2G5o1 TimeInput-module__desktop___1gtms\" placeholder=\"mm\" maxlength=\"2\" size=\"2\" name=\"terra-time-minute-input\" type=\"text\" pattern=\"\\d*\" value=\"\">",
              "target": [
                ".TimeInput-module__time-input-minute___1elGu"
              ]
            }
          ]
        }
      ],
      "any": [],
      "failureSummary": "Fix all of the following:\n  Focusable content should be disabled or be removed from the DOM",
      "html": "<div id=\"root\" aria-hidden=\"true\">",
      "impact": "serious",
      "none": [],
      "target": [
        "#root"
      ]
    }
  ],
  "tags": [
    "cat.name-role-value",
    "wcag2a",
    "wcag412"
  ]
}

```

</details>